src/benchmark.cc

```diff
@@ -574,8 +574,14 @@ size_t RunSpecifiedBenchmarks(BenchmarkReporter* display_reporter,
       std::exit(1);
     }
     if (!file_reporter) {
-      default_file_reporter = internal::CreateReporter(
-          FLAGS_benchmark_out_format, ConsoleReporter::OO_None);
+      if (FLAGS_benchmark_out_format == "console" &&
```

Comment:
why only for console?  i think we should instead just check for the _tabular flag and pass the result to the `CreateReporter` call.

Reply:
That's what I thought afterward, but I considered changing it after feedback. Doing it.